megamek/src/megamek/common/weapons/MissileBayWeaponHandler.java

```diff
@@ -195,56 +194,10 @@ protected double updateAVforAmmo(double current_av, AmmoType atype,
             if (range > WeaponType.RANGE_SHORT) {
                 current_av = 0;
             }
-        }
-        //MMLs get an AV bonus from Artemis V, but not Artemis IV
-        if (atype.getMunitionType() == AmmoType.M_ARTEMIS_V_CAPABLE) {
```

Comment:
Why is all this getting deleted?

Reply:
Because that was an AV hack I put in since bays didn't work. The fixes the mlinker code so it is no longer needed.